packages/@dcl/inspector/src/lib/data-layer/host/utils/component.ts

```diff
@@ -20,3 +22,12 @@ export function findPrevValue(composite: CompositeDefinition, componentName: str
   }
   return value.data.json
 }
+
+export function parseSceneFromComponent(value: DeepReadonlyObject<EditorComponentsTypes['Scene']>) {
+  return {
+    scene: {
```

Comment:
Then add the `description` and `title`

Reply:
yeah, since now we don't provide a way for editing those props, I think we should leave the ones that the `scene.json` already has. We will definitely add those when we update the Scene component